libraries/UHS_host/UHS_USB_IDs.h

```diff
@@ -1921,7 +1921,7 @@
 #define UHS_VID_AACOM 0x0fb9U // AACom Corp.
 #define UHS_VID_SAN_SHING_ELECTRONICS 0x0fbaU // San Shing Electronics Co., Ltd
 #define UHS_VID_BITWISE_SYSTEMS 0x0fbbU // Bitwise Systems, Inc.
-#define UHS_VID_MITAC_INTERNATINAL 0x0fc1U // Mitac Internatinal Corp.
```

Comment:
Rejected, This is an automatically generated file from internet sources

Reply:
Where is the upstream so I can let them know of the typo?

Reply:
Technically, I don't think it actually is. 
I believe it is a non-English spelling. The URL can be found in the python script. 
There's also a notice at the top of the generated header that says that it is automatically generated, and what to run to update.
You can find the URL here:
https://github.com/felis/UHS30/blob/master/libraries/UHS_host/make-USB_IDs.py#L71

Reply:
I pushed a patch via email per the instructions and got this reply. So we'll see if this changes upstream? 

```
This is an automatic reply from the USB ID Mail Robot. If you want to contact
the administrator of the robot, please write to linux.usb.ids+err@gmail.com.

Your submission has been accepted.

--- Processing Log ---

Got mail from luzpaz <luzpaz@foo.bar>, will reply to luzpaz@foo.bar.
Scanning mail for patch.
Patch found.
Searching for original usb.ids version.
Trying /home/usb.ids//origs/usb.ids
patching file usb.ids (read from /home/usb.ids//origs/usb.ids)
Hunk #1 succeeded at 15399 with fuzz 1.
Patch succeeded.
Parsing patched file.
Finding ID differences.
0fc1	Mitac International Corp.
Done.

--- End ---
```